pinot-controller/src/main/java/com/linkedin/pinot/controller/api/restlet/resources/PinotSegmentUploadRestletResource.java

```diff
@@ -428,8 +428,20 @@ private Representation uploadSegment(File indexDir, File dataFile, String downlo
       throws ConfigurationException, IOException, JSONException {
     final SegmentMetadata metadata = new SegmentMetadataImpl(indexDir);
     final File tableDir = new File(baseDataDir, metadata.getTableName());
+    String tableName = metadata.getTableName();
     File segmentFile = new File(tableDir, dataFile.getName());
-    StorageQuotaChecker.QuotaCheckerResponse quotaResponse = checkStorageQuota(indexDir, metadata);
+    OfflineTableConfig offlineTableConfig = (OfflineTableConfig) ZKMetadataProvider
```

Comment:
done

On Thu, Aug 11, 2016 at 3:39 PM, Subbu Subramaniam <notifications@github.com
> wrote:

> In pinot-controller/src/main/java/com/linkedin/pinot/
> controller/api/restlet/resources/PinotSegmentUploadRestletResource.java
> <https://github.com/linkedin/pinot/pull/382#discussion_r74516872>:
>
> >      File segmentFile = new File(tableDir, dataFile.getName());
> > -    StorageQuotaChecker.QuotaCheckerResponse quotaResponse = checkStorageQuota(indexDir, metadata);
> > +    OfflineTableConfig offlineTableConfig = (OfflineTableConfig) ZKMetadataProvider
>
> Can u please add a TODO there? thanks
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/linkedin/pinot/pull/382/files/003b3c0b8cf909de798307844aab085697aa11c3#r74516872>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABAmRq1psirUSSQNfX6DwfJzp-xfnJV7ks5qe6S1gaJpZM4JimHO>
> .
>